src/serializer/Referentializer.js

```diff
@@ -60,7 +60,7 @@ export class Referentializer {
   _referentializedNameGenerator: NameGenerator;
 
   getStatistics(): SerializerStatistics {
```

Comment:
Even better would be to also change this method name to getSerializerStatistics.